external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -22,12 +22,12 @@
     <!-- Coverlet related properties -->
     <CoverletPackageId>coverlet.msbuild</CoverletPackageId>
     <CoverletPackageVersion>2.3.0</CoverletPackageVersion>
-    <!-- In case the project language is not set -->
-    <Language Condition="'$(Language)' == ''">unknown</Language>
+    <!-- Explicitly set Language to c# to handle the analyzer resolution in nuget -->
+    <Language Condition="'$(Language)' == ''">c#</Language>
```

Comment:
Is this still true? We're now going to inject the analyzers directly from the generated props file in Tools.props

Reply:
Still we PackageReference xunit.analyzers here which needs the Language to be set.

Reply:
We can remove this reference and included into Tools.props so that the analyzers actually get injected to the compiler. Due to the infrastructure I added to the repo it has to be there. Also, they're not doing anything in this project, because this project is restored, but when building the tests that already happened, I don't see the xunit analyzers being passed to the compiler.

However this can be fixed in a follow up PR when we enable the analyzers for test projects.

Reply:
> Also, they're not doing anything in this project

Right. We restore xunit.analyzers only as a dependency of xunit. We don't use it today.

> However this can be fixed in a follow up PR when we enable the analyzers for test projects.

I agree.